src/chia_log/handlers/daily_stats/stats_manager.py

```diff
@@ -36,10 +39,10 @@ class StatsManager:
     with a summary from all stats that have been collected for the past 24 hours.
     """
 
-    def __init__(self, config: dict, notify_manager: NotifyManager):
-        self._enable = config.get("enable", False)
-        self._notify_time = self._parse_notify_time(config.get("time_of_day", "21:00"))
-        self._frequency_hours = config.get("frequency_hours", 24)
+    def __init__(self, config: confuse.core.Configuration, notify_manager: NotifyManager):
+        self._enable = config["enable"].get()
```

Comment:
Would be cool to make use of the [validation](https://confuse.readthedocs.io/en/latest/usage.html#validation) that `confuse` provides. Here maybe `get(bool)`.